crates/rolldown_common/src/inner_bundler_options/types/treeshake.rs

```diff
@@ -1,6 +1,11 @@
 use crate::types::js_regex::HybridRegex;
+#[cfg(feature = "deserialize_bundler_options")]
+use schemars::JsonSchema;
+#[cfg(feature = "deserialize_bundler_options")]
+use serde::{Deserialize, Deserializer};
 
 #[derive(Debug)]
+#[cfg_attr(feature = "deserialize_bundler_options", derive(Deserialize, JsonSchema))]
 pub enum TreeshakeOptions {
   False,
```

Comment:
Not for this PR. But We don't need to align the `False` with the rollup's API. Adding filed like `disabled` in `InnerOptions` and using `InnerOptions` directly as the `TreeshakeOptions` won't be a problem.

We should considering if we expose rolldown as a crate, are the exposed options friendly to users.

Reply:
I also think
```rust
enum TreeshakeOptions {
  Bool(bool),
  Option(InnerOptions),
}
```
it fine.

Even though `true` might be meaningless. But considering adde code for dealing `False`, meaningless `true` is not a big deal.

Reply:
Yeah. I agree with you, let's keep the this at the pr, if we need to refactor the API open a new pr for it. cc @IWANABETHATGUY